netjsonconfig/backends/base/backend.py

```diff
@@ -49,6 +50,17 @@ def __init__(self, config=None, native=None, templates=None, context=None):
             raise ValueError('Expecting either config or native argument to be '
                              'passed during the initialization of the backend')
 
+    @classmethod
+    def add_plugins(cls, plugins):
+        for plugin in plugins:
+            plugin.schema.pop('$schema', None)
+            if getattr(cls, 'schema'):
+                cls.schema['properties'][plugin.key] = plugin.schema
+                if getattr(cls, 'converters'):
+                    cls.converters.append(plugin.converter)
```

Comment:
I think it's the same here, what's the use case of not having any converter declared? Wouldn't it just be wrong?